llvm/lib/DebugInfo/Symbolize/SymbolizableObjectFile.cpp

```diff
@@ -311,7 +322,7 @@ DIInliningInfo SymbolizableObjectFile::symbolizeInlinedCode(
                                FileName)) {
       DILineInfo *LI = InlinedContext.getMutableFrame(
           InlinedContext.getNumberOfFrames() - 1);
-      LI->FunctionName = FunctionName;
+      LI->SymbolTableFunctionName = FunctionName;
```

Comment:
Why was this change made?

Reply:
LineInfo doesn't have FunctionName this was needed in order to compile